### PR TITLE
docs: improve accessibility of tohpt4

### PR DESCRIPTION
**Pull request by kapunahelewong**
Fixes contrast issues.
Makes labels explicitly reference inputs.
Removes/replaces "simple" wording.
Adds padding to input so it is an easier target for touch or pointer.
Changes app to application.
Changes "name: " label to "Hero name: ".

### CODE REVIEW

File: aio/content/examples/toh-pt4/src/app/heroes/heroes.component.css [hunk 1]
```diff
@@ -16,17 +16,21 @@
...
-  color: #607D8B;
-  background-color: #DDD;
+  color: #2c3a41;
+  background-color: #e6e6e6;
   left: .1em;
 }
 .heroes li.selected {
-  background-color: #CFD8DC;
-  color: white;
+  background-color: #000;
```
Comment (jelbourn) [inline aio/content/examples/toh-pt4/src/app/heroes/heroes.component.css:24]: Prefer using `black` to `#000` and `white` to `#fff`

**Review by jelbourn — APPROVED** [review]
LGTM

**Review by TeriGlover — APPROVED** [review]
It LGTM.

**Review by aikithoughts — APPROVED** [review]
LGTM!
Reviewed-for: global-docs-approvers

**Pull request closed without merging**